src/settings.py

```diff
@@ -14,6 +14,7 @@
 class GlobalSettings:
     """Unchanging settings."""
     # parameters for all envs
+    num_train_tasks = 50
```

Comment:
do we need to make an announcement about this? it affects amber/ashay

Reply:
yeah